packages/hydrogen/src/Numbers/UIStaticNumber.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { Text, StyleSheet } from 'react-native';
+
+import { uiLocalized } from '@tonlabs/uikit.localization';
+
+import { Typography } from '../Typography';
+import { useTheme, ColorVariants } from '../Colors';
+
+import { useNumberValue } from './useNumberValue';
+import {
+    runOnUILocalizedNumberFormat,
+    UINumberDecimalAspect,
+} from './runOnUILocalizedNumberFormat';
+import type { UINumberAppearance, UINumberGeneralProps } from './types';
+
+export function useNumberStaticStyles(integerColor: ColorVariants, decimalColor: ColorVariants) {
+    const theme = useTheme();
+    const integerColorStyle = React.useMemo(
+        () => ({ color: theme[integerColor] }),
+        [theme, integerColor],
+    );
+    const decimalColorStyle = React.useMemo(
+        () => ({ color: theme[decimalColor] }),
+        [theme, decimalColor],
+    );
+
+    return [integerColorStyle, decimalColorStyle];
+}
+
+export function UIStaticNumber({
+    testID,
+    children,
+    integerVariant,
+    integerColor,
+    decimalVariant,
+    decimalColor,
+    decimalAspect = UINumberDecimalAspect.None,
+    sign,
+}: UINumberGeneralProps & UINumberAppearance & { sign?: React.ReactNode }) {
+    const value = useNumberValue(children, decimalAspect);
+    const { decimal: decimalSeparator, grouping: integerGroupChar } =
+        uiLocalized.localeInfo.numbers;
+
+    const formatted = React.useMemo(() => {
+        return runOnUILocalizedNumberFormat(
+            value,
+            decimalAspect,
+            decimalSeparator,
+            integerGroupChar,
+        );
+    }, [value, decimalAspect, decimalSeparator]);
+
+    const [integerColorStyle, decimalColorStyle] = useNumberStaticStyles(
+        integerColor,
+        decimalColor,
+    );
+
+    return (
+        <Text testID={testID} accessibilityLabel={`${formatted.integer}${formatted.decimal}`}>
```

Comment:
Please add '.' between formatted.integer and formatted.decimal

Reply:
It's [already there](https://github.com/tonlabs/UIKit/pull/1285/files#diff-0c828d2707549d5e6e6129d9104fb10012b0a3625e50ca3e8b5daa9ebbf128a7R113) :)

Reply:
great, thanks!